docs/static/breaking-changes.asciidoc

```diff
@@ -34,20 +34,45 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
 * Configurations provided with `-f` or `config.path` will not be appended with `stdin` input and `stdout` output automatically.
 
 [float]
+=== Plugin Changes
 
+[float]
 ==== Elasticsearch output changes
 
-* The default `document_type` has changed from `logs` to `doc`.
+* The default `document_type` has changed from `logs` to `doc` for consistency with Beats.
   Furthermore, users are advised that Elasticsearch 6.0 deprecates doctypes, and 7.0 will remove them. 
   See https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html[Removal of Mapping Types] for more info.
 * The options `flush_size` and `idle_flush_time` are now obsolete.
 * Please note that the https://www.elastic.co/guide/en/elasticsearch/reference/6.0/mapping-all-field.html[_all] field is deprecated in 6.0.
  The new mapping template has been updated to reflect that. If you are using a custom mapping template you may need to update it to reflect that.
 
 [float]
+==== Kafka input changes
+
+* Upgrade Kafka client to version 0.11.0.0. Only supports brokers 0.10.1.x or later.
+** Please refer to <<plugins-input-kafka>> documentation for information about Kafka compatibility with Logstash.
+* Decorated fields are now nested under `@metadata` to avoid mapping conflicts with Beats.
+** See the `Metadata Fields` section in the <<plugins-input-kafka>> documentation for more details.
+* The `ssl` option is now obsolete.
+
+[float]
+==== Kafka output changes
+
+* Upgrade Kafka client to version 0.11.0.0. Only supports brokers 0.10.1.x or later.
+** Please refer to <<plugins-input-kafka>> documentation for information about Kafka compatibility with Logstash.
```

Comment:
So that the link resolves to the text you expect, you should say "Please refer to the <<plugins-input-kafka,Kafka input plugin>> documentation....."

This is necessary because we use abbreviated titles to make the navigation more readable, but it changes the link text, too.